doc/nrf/doc_styleguide.rst

```diff
@@ -57,6 +57,16 @@ It is also possible to define more than one default link text for a link, which
    .. _`Link text one`:
    .. _`Link text two`: http://..
 
+Breathe Domains
+===============
+
+In order to link to Doxygen content from RST, the following domains are to
+be used:
+
+* Function: ``:cpp:func:``
+* Structure: ``:c:type:``
+* Enum: ``:cpp:enumerator:``
+* Macro: ``:c:macro:``
 
 Doxygen style guide
```

Comment:
Should :option: (for Kconfig options) be mentioned here?

Reply:
It's not really part of breathe, but I can reword it so that it's present